src/frontend/running.rs

```diff
@@ -52,21 +66,27 @@ struct FarmerState {
     token_symbol: String,
 }
 
+#[allow(dead_code)]
 #[derive(Debug)]
 pub struct RunningView {
     node_view: Controller<NodeView>,
     node_synced: bool,
     farmer_state: FarmerState,
     farms: FactoryHashMap<u8, FarmWidget>,
     plotting_paused: bool,
+    slot_probability: (u64, u64),
+    space_pledged: u128,
+    last_reward_timestamp: Option<u64>,
```

Comment:
These seem to belong to `FarmerState` rather than `RunningView`. With growing view file we'll likely move farmer's presentation into a separate file and its properties should naturally move alongside it.